doc/model_persistence.rst

```diff
@@ -181,3 +267,29 @@ not help in production when performance is critical.
 To convert scikit-learn model to PMML you can use for example `sklearn2pmml
 <https://github.com/jpmml/sklearn2pmml>`_ distributed under the Affero GPLv3
 license.
+
+Summarizing the keypoints
+-------------------------
+
+Based on the different approaches for model persistence, the keypoints for each
+approach can be summarized as follows:
+
+* `pickle`: It is native to Python and any Python object can be serialized and
+  deserialized using `pickle`, including custom Python classes and objects.
+  While `pickle` can be used to easily save and load scikit-learn models,
+  unpickling of untrusted data might lead to security issues.
+* `joblib`: Efficient storage and memory mapping techniques make it faster
+  for serializing and deserializing large numpy arrays. However, care must be
+  taken when loading untrusted data as it can lead to the execution of
+  arbitrary Python code, which may be malicious.

```

Comment:
```suggestion
  when working with large machine learning models or large numpy arrays. However,
  it may trigger the execution of malicious code while loading untrusted data.
```